src/utils/constant/headerLink.ts

```diff
@@ -0,0 +1,13 @@
+import { ILink } from '../../elements/FooterLinkGroup/ui/index';
```

Comment:
```suggestion
import type { ILink } from '../../elements/FooterLinkGroup/ui/index';
```

Лучше помечать типы в импортах и реэкспортах. А еще, если это какой-то универсальный тип, его наверное можно в utils отправить, что не было импорта в headerLink и footerLink 😺